src/storage/src/hummock/event_handler/refiller.rs

```diff
@@ -294,29 +310,107 @@ impl CacheRefillTask {
         let mut tasks = vec![];
         for sst_info in &holders {
             let task = async move {
-                GLOBAL_CACHE_REFILL_METRICS.data_refill_attempts_total.inc();
+                if let Err(e) = Self::data_file_cache_refill_impl(context, sst_info.value()).await {
+                    tracing::warn!("data cache refill error: {:?}", e);
+                }
+            };
+            tasks.push(task);
+        }
 
-                let permit = context.concurrency.acquire().await.unwrap();
+        join_all(tasks).await;
+    }
 
-                GLOBAL_CACHE_REFILL_METRICS.data_refill_started_total.inc();
+    async fn data_file_cache_refill_impl(
```

Comment:
How about add some comments for the new refill strategy with unit-level ?

Reply:
There are comments on the cache refill config about unit-level cache refill. But I agree with you, more details are needed here.